tests/test_routes.py

```diff
@@ -1,5 +1,6 @@
 from db.models.assessment_record.assessment_records import AssessmentRecord
 from tests._helpers import get_random_row
+from tests._helpers import get_rows_by_filters
 
 
 def test_gets_all_apps_for_fund_round(request, client):
```

Comment:
Can we move away from randomly generated test data please? Makes the tests not repeatable and much harder to debug :)